clients/nexus-iq/src/main/kotlin/NexusIqService.kt

```diff
@@ -42,6 +42,18 @@ import retrofit2.http.Path
  */
 interface NexusIqService {
     companion object {
+        /** Identifier of the scoring system _Common Vulnerability Scoring System_ version 2. */
+        const val SCORE_CVSS2 = "CVSS2"
+
+        /** Identifier of the scoring system _Common Vulnerability Scoring System_ version 3. */
+        const val SCORE_CVSS3 = "CVSS3"
+
+        /**
+         * A Sonatype-specific prefix for references of security issues. The prefix determines how some of the
+         * properties need to be interpreted, e.g. the severity value.
+         */
+        const val PREFIX_SONATYPE = "sonatype-"
```

Comment:
Sorry, one more nit: In the ORT code base, we would name this `SONATYPE_PREFIX` instead. Similar for the above constants, we would use `CVSS2_SCORE` etc.